src/screens/NewsScreen/index.js

```diff
@@ -271,7 +271,7 @@ const NewsScreen = () => {
         screen="news_screen"
         getValueBlock={onBlockedDomain}
       />
-    </SafeAreaProvider>
+    </SafeAreaView>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The closing tag for the root container has been changed from `SafeAreaProvider` to `SafeAreaView`. This is consistent with the opening tag change and ensures that the JSX syntax is correct.
```diff
-    </SafeAreaProvider>
+    </SafeAreaView>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->